src/frog-base.tsx

```diff
@@ -467,33 +500,6 @@ export class FrogBase<
       )
     })
 
-    // OG Image Route
-    this.hono.get(`${parsePath(path)}/image`, async (c) => {
-      const defaultImageOptions = await (async () => {
-        if (typeof this.imageOptions === 'function')
-          return await this.imageOptions()
-        return this.imageOptions
-      })()
-
-      const fonts = await (async () => {
-        if (typeof options?.fonts === 'function') return await options.fonts()
-        if (options?.fonts) return options.fonts
-        return defaultImageOptions?.fonts
-      })()
-
-      const {
-        headers = this.headers,
-        image,
-        imageOptions = defaultImageOptions,
-      } = fromQuery<any>(c.req.query())
-      const image_ = JSON.parse(lz.decompressFromEncodedURIComponent(image))
-      return new ImageResponse(image_, {
-        ...imageOptions,
-        fonts: await parseFonts(fonts),
-        headers: imageOptions?.headers ?? headers,
-      })
-    })
-
```

Comment:
moved this upper to ensure the routing priority is preserved

https://hono.dev/api/routing#routing-priority